install.sh

```diff
@@ -89,6 +89,29 @@ if [ -n "$pkg" ] && [ "$pkg" != "server" -a "$pkg" != "conf" -a "$pkg" != "kerne
     exit 1
 fi
 
+patchelf() {
+    # patchelf comes from the build system, so it needs the build system's ld.so and
+    # shared libraries. We can't use patchelf on patchelf itself, so invoke it via
+    # ld.so.
+    LD_LIBRARY_PATH="$PWD/libreloc" libreloc/ld.so libexec/patchelf "$@"
+}
+
+adjust_bin() {
+    local bin="$1"
+    # We could add --set-rpath too, but then debugedit (called by rpmbuild) barfs
+    # on the result. So use LD_LIBRARY_PATH in the thunk, below.
```

Comment:
Do we know why? What is the error?

Reply:
The error manifests in debugedit complaining that DWARF version 0 is unsupported, followed by another failure. Random googling shows a bug in patchelf where the linker used the rpath string as a string pool and pointed other strings into it (not 100% sure), and since patchelf nuked it those other strings got corrupted. Maybe it's related and maybe not.